infrastructure/azure.tf

```diff
@@ -998,6 +1000,48 @@ resource "azurerm_dns_cname_record" "azurerm_dns_main_zone_dkim" {
   ttl                 = 3600
 }
 
+#
+# Redis cache
+# 
+resource "azurerm_storage_account" "azurerm_redis_backup" {
+  name                      = "${local.azurerm_redis_backup_name}"
+  resource_group_name       = "${azurerm_resource_group.azurerm_resource_group.name}"
+  location                  = "${azurerm_resource_group.azurerm_resource_group.location}"
+  account_tier              = "Standard"
+  account_replication_type  = "GRS"
+  enable_blob_encryption    = true
+  enable_https_traffic_only = true
+}
+
+resource "azurerm_redis_cache" "azurerm_redis_cache" {
+  name                = "${local.azurerm_redis_cache_name}"
+  location            = "${azurerm_resource_group.azurerm_resource_group.location}"
+  resource_group_name = "${azurerm_resource_group.azurerm_resource_group.name}"
+
+  # Possible values for Premium: 1, 2, 3, 4
```

Comment:
link to docs, what does it mean 1, 2, 3...?

Reply:
also, perhaps a short comment describing why we decided to use capacity=1 and shard_count=2